### DISCUSSION TITLE
Use collection entries as form select options

### DISCUSSION THREAD

**Discussion by ozmos** [statamic/cms#5478] [Q&A]
Hi, I'm building a form for competition entries.  I have a collection called `events` from which I'd like participants to choose events from.  What is the standard practice to use collection entries in forms?  Unfortunately I can't share any code as it's for a client.

**Comment by ryanmitchell** [comment]
You could make the entries field type selectable in forms (see: https://statamic.dev/forms#the-blueprint) and then allow entries to be selected in the blueprint.

**Reply by ozmos** [reply]
<del>Thanks, the answer is usually in the docs somewhere. </del>

**Reply by ozmos** [reply]
Unfortunately this didn't allow me to loop through the entries in a collection on the form front end.  I ended up doing a work around with a `collections` field type set on the page and a hidden `entry` field on the form.
